src/client/utils/API.tsx

```diff
@@ -0,0 +1,62 @@
+export {}
+
+async function getBFFServiceUrl(token: string, baseURL: string) {
+    // TODO: for dev, the baseUrl will be whatever works for the CRA workflow, which is ''.
+    // baseURL = 'https://ci.kbase.us/services'; // for dev
+    window.window._env.narrative
```

Comment:
This line does nothing (line 6)
Also, what's up with using `window.window._env` instead of `window._env`?